minutiae/intro.tex

```diff
@@ -112,7 +112,7 @@ \section*{Acknowledgments}
 Borys Pierov \\
 Carise Fernandez \\
 Charles	Stanhope \\
-Chris Atlee \\
+Chris AtLee \\
```

Comment:
I actually triple-checked them, so if I haven't gotten it right by now I don't think I'm going to do any better :)